08-projects.Rmd

```diff
@@ -4046,9 +4048,9 @@ head airports.csv
 
 **Note:** You may be wondering *why* we are asking you to do this. This sort of situation (where you need to deal with quotes) happens a lot! It's important to practice and learn ways to fix these things.
 
-**Hint:** You could use `gsub` within `awk` to replace '"' with ''.
+**Hint:** You could use `gsub` within `awk` to replace '"' with ''. You can find how to use `gsub` [here](https://www.gnu.org/software/gawk/manual/html_node/String-Functions.html).
```

Comment:
There seems no `gsub` explanation in the awk section of the Examples Book. Hence I include a link to the manual.